### PR TITLE
Swiss municipalities mutations audit

### PR DESCRIPTION
**Pull request by ptbrowne**
Audit mutations of municipality

Municipalities in Switzerland change over time. These "mutations" are published as a list here : https://www.agvchapp.bfs.admin.ch/de/mutations/results?EntriesFrom=01.01.2023&EntriesTo=01.01.2024. It can be helpful
to review side by side on a map what was done.

This PR adds a page that provides a side by side comparison of two different years.

- Go to https://swiss-maps-git-mutations-ixt1.vercel.app/mutations

## Comments

- At first the input was coming from a copy/paste of the HTML table from the site. Then I changed to use an API route. The API route loads the aforementioned mutations page and extracts the data from the "<script />". It's a bit hacky but I am not aware of an API.
- It loads the map data from the year of the mutation and the (year - 1) of the mutations.

### CODE REVIEW

File: website/src/components/Generator/internal/Preview.tsx [hunk 1]
```diff
@@ -139,10 +89,15 @@ export const Preview = React.forwardRef(({}: Props, deckRef: any) => {
         // domain is decided by coloring item size
         // currently only support cantons
         // if not exist, a random number 30 is assigned
-        .domain(["1", state.geoData?.cantons?.length ?? "30"])
+        .domain([
+          "1",
+          geoData?.cantons?.features?.length
```
Comment (shuaiey) [inline website/src/components/Generator/internal/Preview.tsx:94]:
```suggestion
          geoData.cantons?.features?.length
```

File: website/src/components/Generator/internal/Preview.tsx [hunk 2]
```diff
@@ -139,10 +89,15 @@ export const Preview = React.forwardRef(({}: Props, deckRef: any) => {
...
+            ? `${geoData?.cantons?.features?.length}`
```
Comment (shuaiey) [inline website/src/components/Generator/internal/Preview.tsx:95]:
```suggestion
            ? `${geoData.cantons.features.length}`
```

**Comment by ptbrowne** [comment]
I did not really know where to put it to be honest and I did not think that having it as the same level as "Examples", "Documentation" etc... would be good, so this is why I put it in examples. Would you put a "Tools" section only with this ?

**Comment by ptbrowne** [comment]
I would consider this again if we have another tool 👍 Thanks for the suggestion.

**Pull request merged**